api_tests/lib_sdk/utils.ts

```diff
@@ -4,6 +4,19 @@ import { SwapRequest } from "comit-sdk";
 import * as fs from "fs";
 import { promisify } from "util";
 
```

Comment:
Can you merge all the imports together by removing this new line and putting the comment on the same line than the import (ie, on the right).